src/content/app/help/components/help-menu/HelpMenu.tsx

```diff
@@ -69,28 +72,42 @@ const HelpMenu = (props: Props) => {
   };
 
   const closeMegaMenu = () => {
+    setActiveTopLevelMenuName('');
     setSubmenuItems(null);
     clickedMenuRef.current = null;
   };
 
-  const topLevelItems = props.menu.items.map((item, index) => {
-    const className = classNames(styles.topMenuItem);
-    const commonProps = {
-      key: index,
-      className
-    };
+  const getTopLevelMenuItemClasses = (menuName: string) =>
+    classNames(styles.topMenuItem, {
+      [styles.topMenuItemActive]: activeTopLevelMenuName === menuName
+    });
+
+  const getHelpMenuLinkClasses = () =>
+    classNames(styles.topMenuItem, {
+      [styles.topMenuItemActive]:
+        !activeTopLevelMenuName &&
+        ['/about', '/help'].includes(props.currentUrl)
+    });
```

Comment:
For the top-level menu items that do not have submenus, you want to mark them as active when the value of their `url` field is the same as the location.pathname. Plus, as an extra check, we want to make sure that no other top-level menu item is currently active, i.e. that  `clickedMenuRef.current === null` — that would allow us to avoid paining the link in the top-menu item black when the user has opened the meganav:

<img width="361" alt="image" src="https://user-images.githubusercontent.com/6834224/234556948-4459591a-b2df-4773-8ca6-50e69ca64606.png">

Reply:
✅